apps/meteor/client/views/room/composer/messageBox/MessageBoxActionsToolbar/hooks/useFileUploadAction.ts

```diff
@@ -30,7 +30,7 @@ export const useFileUploadAction = (disabled: boolean): GenericMenuItemProps =>
 				});
 				return file;
 			});
-			chat?.flows.uploadFiles(filesToUpload, resetFileInput);
```

Comment:
References of chat should be removed since it's not used anymore

Reply:
done sir removed the chat references